build_web_compilers/lib/src/web_entrypoint_builder.dart

```diff
@@ -77,7 +77,8 @@ class WebEntrypointBuilder implements Builder {
             'Only `dartdevc` and `dart2js` are supported.');
     }
 
-    var dart2JsArgs = options.config[_dart2jsArgs] ?? const <String>[];
+    var dart2JsArgs =
+        options.config[_dart2jsArgs].cast<String>() ?? const <String>[];
```

Comment:
needs to be `?.cast<String>()`. Otherwise the `??` isn't doing anything.

Reply:
ah yep good catch